### DISCUSSION TITLE
Newbie help with docker install

### DISCUSSION THREAD

**Discussion by tallmtt** [Kovah/LinkAce#406] [Support] [answered]
I have a server on my domain with other php programs installed without docker. 

I would like to install linkace under my domain.com/linkace 

When I try to install this with docker, I get this error:

`Creating linkace_nginx_1 ... error

ERROR: for linkace_nginx_1  Cannot start service nginx: driver failed programming external connectivity on endpoint linkace_nginx_1 (e08da2e12fd4f4f33fa65673eac39fc9742ccce9d3234b3d6e93d34e84494637): Error starting userland proxy: listen tcp4 0.0.0.0:443: bind: address already in use

ERROR: for nginx  Cannot start service nginx: driver failed programming external connectivity on endpoint linkace_nginx_1 (e08da2e12fd4f4f33fa65673eac39fc9742ccce9d3234b3d6e93d34e84494637): Error starting userland proxy: listen tcp4 0.0.0.0:443: bind: address already in use
ERROR: Encountered errors while bringing up the project`

How can I adjust the port, so I can make this work? I am using Apache2, which binds 443 for my domains.

**Comment by Kovah** [comment, marked as answer]
Hi!
First of all, LinkAce may not correctly work in a subdirectory, it should better be installed with a subdomain (see #264).
I can't help you with your exact setup, but if you already have an Apache web server, you should use it in front of LinkAce. Running an nginx at the same level won't work. You can find a lot of tutorials on how to use Apache as a "proxy". Then, change the ports of nginx, to use e.g. 8443 instead of 443.

**Reply by tallmtt** [reply]
Thanks for the reply - I set it up as a subdomain without any issue.
